features/admin.organizations.v1/components/edit-organization/organization-overview.tsx

```diff
@@ -475,6 +475,25 @@ export const OrganizationOverview: FunctionComponent<OrganizationOverviewPropsIn
                                         minLength={ 3 }
                                     />
                                 ) }
+                                { organization?.orgHandle && (
+                                    <Field.Input
+                                        data-testid={ `${ testId }-overview-form-org-handle-input` }
+                                        name="orgHandle"
+                                        label={ t(
+                                            "organizations:edit.fields.orgHandle.label"
+                                        ) }
+                                        required={ false }
+                                        requiredErrorMessage=""
```

Comment:
If required is false, we wouldn't need to explicitly define these two attributes, IMO.

```suggestion
```